module/builder/collection/builder_test.go

```diff
@@ -823,8 +823,21 @@ func (suite *BuilderSuite) TestBuildOn_NoRateLimiting() {
 
 	// since we have no rate limiting we should fill all collections and in 10 blocks
 	parentID := suite.genesis.ID()
+	setter := func(h *flow.HeaderBodyBuilder) error {
+		h.WithHeight(1).
+			WithChainID(flow.Emulator).
+			WithParentID(unittest.IdentifierFixture()).
```

Comment:
Was this intended to be `parentID`?
```suggestion
			WithParentID(parentID).
```